### PR TITLE
docs: update v2 README

### PR DESCRIPTION
**Pull request by w3b6x9**
## What kind of change does this PR introduce?

docs update

## What is the new behavior?

Update the README to reflect the new v2 API.

## Additional Context

See https://github.com/supabase/realtime-js/tree/docs/readme-v2 for README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,128 +1,213 @@
...
+  <h1 align="center">Supabase Realtime Client</h1>
 
-This is for usage with Supabase [Realtime](https://github.com/supabase/realtime) server.
+  <h3 align="center">Send ephemeral messages with <b>Broadcast</b>, track and synchronize state with <b>Presence</b>, and listen to database changes with <b>Postgres Change Data Capture (CDC)</b>.</h2>
 
-## Usage
+  <p align="center">
+    <a href="https://supabase.com/docs/guides/realtime">Guides</a>
+    ·
+    <a href="https://supabase.com/docs/reference/javascript">Documentation</a>
```
Comment (dannykng) [inline README.md:18]: Maybe "Reference docs" so it's a bit clearer how it's different from guides
```suggestion
    <a href="https://supabase.com/docs/reference/javascript">Reference docs</a>
```

File: README.md [hunk 2]
```diff
@@ -1,128 +1,213 @@
...
+      <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/supabase/supabase/master/packages/common/assets/images/supabase-logo-wordmark--light.svg">
+      <img alt="Supabase Logo" width="300" src="https://raw.githubusercontent.com/supabase/supabase/master/packages/common/assets/images/logo-preview.jpg">
+    </picture>
+  </a>
 
-Listens to changes in a PostgreSQL Database and via websockets.
+  <h1 align="center">Supabase Realtime Client</h1>
 
-This is for usage with Supabase [Realtime](https://github.com/supabase/realtime) server.
+  <h3 align="center">Send ephemeral messages with <b>Broadcast</b>, track and synchronize state with <b>Presence</b>, and listen to database changes with <b>Postgres Change Data Capture (CDC)</b>.</h2>
```
Comment (dannykng) [inline README.md:13]: Did you want this as an h3 or an h2 (different opening and closing tags)?

File: README.md [hunk 3]
```diff
@@ -1,128 +1,213 @@
...
-- `realtime:{schema}` - where `{schema}` is the Postgres Schema
-- `realtime:{schema}:{table}` - where `{table}` is the Postgres table name
-- `realtime:{schema}:{table}:{col}=eq.{val}` - where `{col}` is the column name, and `{val}` is the value which you want to match
+- `REALTIME_URL` is `'ws://localhost:4000/socket'` when developing locally and `'wss://<project_ref>.supabase.co/realtime/v1'` when connecting to your Supabase project.
+- `API_KEY` is a JWT whose claims must contain `exp` and `role` (existing database role).
+- Channel name can be any `string`.
+- `eventsPerSecond`, or client-side rate limiting, enforces the number of events sent to the Realtime server uniformly spread across a second. The default is 10, which means that the client can send one event, whether that's **Broadcast**/**Presence**/**Postgres CDC**, every 100 milliseconds. You may change this as you see fit but note that the server's rate limiting will need to be updated accordingly. You can learn more about Realtime's rate limits here: https://supabase.com/docs/guides/realtime/rate-limits.
 
-**Examples**
+### Broadcast
```
Comment (dannykng) [inline README.md:80]: nit: Should this be an h2 instead (along with all the following headings that aren't notes)? Assuming the notes are specific to the preceding heading...

Example
```
- Installing the package
  - Notes
- Broadcast
  - Notes
- Presence
- Postgres CDC
...
```

**Review by dannykng — APPROVED** [review]
Left a few comments but LGTM!

File: README.md (hunk 2 above)
Comment (w3b6x9) [reply README.md:13]: @dannykng ah, thanks! i meant h3

Comment (w3b6x9) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (w3b6x9) [reply README.md]: @dannykng yep, good call. updated!

**Pull request merged**